content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,241 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+Sourcegraph 4.0, the latest version of our code intelligence platform, is now available. Our code intelligence platform includes a dozen new features to grok code, spend more time in flow, and execute big decisive refactors across your codebase.
+
+A code intelligence platform indexes and understands all of your organization's code, no matter where it's stored or what language it's in. With Sourcegraph, devs and engineering leaders have a complete understanding of their codebase, so they can learn about code and operate more efficiently. 
+
+Read on to discover everything that shipped in Sourcegraph 4.0.  
+
+### 🧠 Code intelligence: uplevel your code search
+
+<br />
+<a href="/code-insights"><Badge text="Code Insights" color="green" size="small" /></a>
+
+#### Understand usage and search structure with Code Insights dashboards within the search UI
```

Comment:
@Joelkw Could you please review this by EOD today to ensure the content is accurate? Please keep in mind that we're focusing on the benefits and value to our customers/prospects within this release post.